pkg/collector/check/stats.go

```diff
@@ -11,13 +11,21 @@ import (
 
 	"github.com/DataDog/datadog-agent/pkg/telemetry"
 	telemetry_utils "github.com/DataDog/datadog-agent/pkg/telemetry/utils"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+	"github.com/mitchellh/mapstructure"
 )
 
 const (
 	runCheckFailureTag = "fail"
 	runCheckSuccessTag = "ok"
 )
 
+// EventPlatformNameTranslations contains human readable translations for event platform event types
+var EventPlatformNameTranslations = map[string]string{
```

Comment:
See my comment https://github.com/DataDog/datadog-agent/pull/7735#discussion_r613258957 

I don't think a subsection will provide a meaningful grouping as this "event platform" grouping is more of an implementation detail than a useful user-facing concept. 

We need the translation otherwise it's going to look wrong, since all the other types of telemetry are effectively "translated" to more friendly names:

* `'service_checks' -> 'Service Checks'` 
* `'metrics' -> 'Metrics Samples'`